src/utils.py

```diff
@@ -29,9 +34,13 @@ def log_entry(log_file: OutputFile, message: str) -> None:
         log_file: log file.
         message: log message.
     """
-    with open(log_file.path, "a", encoding="utf-8") as log_writer:
-        log_writer.write(message)
-        log_writer.write("")
+    try:
+        with open(log_file.path, "a", encoding="utf-8") as log_writer:
+            log_writer.write(message)
+            log_writer.write("\n")
+    except Exception as e:
+        logger.error("Failed to write to log file %s: %s", log_file.path, e)
+        logger.info("Original log message: %s", message)
 
 
 def mount_disk(image_path: str, mount_point: str) -> str:
```

Comment:
openworker_common_lib has mount functionality available now.
https://github.com/openrelik/openrelik-worker-common/blob/main/openrelik_worker_common/mount_utils.py

See if it is easy to replace and useful for you or open an issue to look into this (so we do not forget). Replace worker mounting methods with openrelik_worker_common mounting functionality.

Reply:
I considered using `openrelik_worker_common/mount_util.py`. I noticed mount path uses full uuid in `mount_util.py`[0], which is 32 chars long. Container Explorer has limitation on the length of mount string. Using full uuid4 length will increase the possibility of running into the mount command length issue.

I will create a issue for now and we can revisit this once we have options to use shorter mount path in `openrelik_worker_common/mount_util.py`.

[0] https://github.com/openrelik/openrelik-worker-common/blob/main/openrelik_worker_common/mount_utils.py#L420.

Reply:
https://github.com/openrelik/openrelik-worker-common/issues/37